compiler/rustc_target/src/spec/targets/hexagon_unknown_qurt.rs

```diff
@@ -0,0 +1,50 @@
+use crate::spec::{Cc, LinkerFlavor, Target, TargetMetadata, TargetOptions, cvs};
+
+pub(crate) fn target() -> Target {
+    let mut pre_link_args = std::collections::BTreeMap::<LinkerFlavor, Vec<_>>::new();
+    pre_link_args
+        .entry(LinkerFlavor::Unix(Cc::Yes))
+        .or_default()
+        .extend(["-G0".into()].into_iter());
```

Comment:
```suggestion
        .extend(["-G0".into()]);
```

I don't think the explicit `.into_iter()` is necessary.